keras/layers/core.py

```diff
@@ -863,7 +865,8 @@ def build(self, input_shape):
                                       initializer=self.kernel_initializer,
                                       name='kernel',
                                       regularizer=self.kernel_regularizer,
-                                      constraint=self.kernel_constraint)
+                                      constraint=self.kernel_constraint,
+                                      sparse_weight=self.sparse_weight)
```

Comment:
what would happen here for tensorflow backend if Dense if initialized as Dense(sparse_weight=True)?
Shall we also add backend check here?